README.md

```diff
@@ -124,3 +128,5 @@ Grater: https://openclipart.org/detail/180156/grater
 Pineapple Pie: http://res.freestockphotos.biz/pictures/17/17487-illustration-of-a-slice-of-pie-pv.png (Resized)
 
 Whisk: https://pixabay.com/en/mixer-electric-appliance-equipment-31112/
+
+Fork and Spatula: Patrick Wang
```

Comment:
and this is the other. I wanted you to remove one of them.
But I guess I'll do it myself now!